.changelog/1425.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+resource/cloudflare_api_token: Avoid recreating the resources when multiple `permission_groups` exist.
```

Comment:
i think this is slightly confusing as we're not fixing multiple cases of `permission_groups` but instead, ignoring the ordering change in the permission IDs that are returned from the API.

```suggestion
resource/cloudflare_api_token: ignore ordering of `permission_group` IDs
```